src/CitizenProject/CitizenProjectBroadcaster.php

```diff
@@ -55,9 +59,23 @@ private function createMessage(array $citizenProjects, Adherent $adherent): Mess
             ]
         );
 
+        $urls = [
+            'all_citizen_projects_url' => $this->urlGenerator->generate(
+                'app_search_citizen_projects',
+                [],
+                UrlGeneratorInterface::ABSOLUTE_URL
+            ),
+            'email_notifications_url' => $this->urlGenerator->generate(
+                'app_user_set_email_notifications',
+                [],
+                UrlGeneratorInterface::ABSOLUTE_URL
+            ),
+        ];
+
         return CitizenProjectApprovedSummaryMessage::create(
             $adherent,
-            $summary
+            $summary,
+            $urls
```

Comment:
keys `all_citizen_projects_url` and `email_notifications_url` belongs to the EmailTemplate logic, not the broadcaster.

can you please send 2 distincts `string` variables for these dynamic url?